README.md

```diff
@@ -11,20 +11,15 @@ The data is pulled from two different sources:
 
 A web tool to explore the data is available [here](https://telemetry.mozilla.org/probe-dictionary/).
 
+CI on this repository handles publishing the data to populate the `probeinfo` API
+and the related [API docs](https://mozilla.github.io/probe-scraper/):
+
+![probeinfo API docs](docs.png)
+
 ## Adding a New Glean Repository
 
 To scrape a git repository for probe definitions, an entry needs to be added in `repositories.yaml`.
-
-- `description`: A brief description of the repository.
-- `channel`: (optional) The release channel. Must be one of "release", "beta", "nightly" or "esr".
-- `deprecated` (optional) `true` if repository corresponds to a deprecated product or library.
-- `prototype`: (optional) `true` if the application is still under developement.
-- `notification_emails`: Where emails about probe-scraper failures and improper files will be forwarded to. These
-will be just about your specific repository.
-- `url`: The URL of the repository to scrape. It should be able to be cloned directly from that URL.
-- `branch` (optional): The branch in the repository to use. (Defaults to `master`).
-- `metrics_files`: A list of relative paths to `metrics.yaml` files
-- `ping_files`: A list of relative paths to `pings.yaml` files
+Refer the the [schema docs](https://mozilla.github.io/probe-scraper/#operation/getReposYaml).
```

Comment:
As mentioned, I'm not sure about the OpenAPI stuff, but if we're going to use it I would rewrite this to:

```suggestion
For a full list of fields that need to be added (and what they mean), see the [schema documentation](https://mozilla.github.io/probe-scraper/#operation/getReposYaml).
```